mesh/fake-mesh.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
```

Comment:
Could you just provide an env.example.sh for fake-mesh instead of duplicating the script?